res/database/Default/user/scr0llbaer.xml

```diff
@@ -467,30 +510,121 @@ Format: Filmed lectures. Laid-back and to lay down.</en>
 				<modifier name="topicality::age" value="5" />
 				<modifier name="topicality::refresh" value="2" />
 			</programme_data_modifiers>
-		</scripttemplate-->
+		</scripttemplate>
 
-		<!--scripttemplate product="3" licence_type="1" guid="9f1cacc4-aa56-4f4a-9a9f-8c21c4bd3a7d" creator="9551" comment="Perhaps some reciprocal effect with the corresponding news genre can be implemented? TODO: I am not happy with the titles and descriptions of these.">
+		<scripttemplate product="3" licence_type="1" guid="56bcb1ff-8a5d-4d91-b3f7-ad6958b3855b" creator="9551">
+			<title>
+				<de>%recreational%%athletes% aus %randomcity% und anderswo</de>
+				<en>%recreational% %athletes% from %randomcity% and elsewhere</en>
+			</title>
+			<description>
+				<de>Aktuelle News und Berichterstattungen aus den Bunt-, Amateur- und Kreisligen der Nation. Fußball, Eishockey, Basketball, Football, Tennis, Tischtennis, Leichtathletik, Kirmesboxen, Kartsport und Formel 3 &amp; 4, und vieles mehr.</de>
+				<en>Latest news and coverage from the nation's hobby, amateur and county leagues. Soccer, ice hockey, basketball, football, tennis, athletics, fun fair boxing, karting and Formula 3 &amp; 4, and much more.</en>
+			</description>
+
+			<variables>
+				<recreational>
+					<de>Freizeit|Feierabend|Hobby|Amateur</de>
+					<en>Recreational|Leisure|Hobby|Amateur</en>
+				</recreational>
+				<athletes>
+					<de>athleten|kicker und co.|-Sportfreunde</de>
+					<en>Athletes|Footballers and co.|Sports Friends</en>
+				</athletes>
+				<randomcity>
+					<de>%STATIONMAP:RANDOMCITY%</de>
+					<en>%STATIONMAP:RANDOMCITY%</en>
+				</randomcity>
+			</variables>
+
+			<jobs>
+				<job index="0" function="1" required="1" />
+				<job index="1" function="4" required="1" />
+				<job index="2" function="8" required="1" />
+				<job index="3" function="128" required="1" />
+				<job index="4" function="64" required="0" />
+			</jobs>
+			<data flags="1" production_limit="100" production_broadcast_limit="2" production_broadcast_flags="512" production_licence_flags="36" />
```

Comment:
It's like a sports magazine, it run indefinitely.

Perhaps a nice spin would be there was a way to configure it to make it display a different `%randomcity%` with each "episode" (and not just with each script instantiation)?

Reply:
Currently, this is not possible (I don't know what happens, if you put %randomcity% directly into the title - the names should differ in different languages, but my guess is that they won't differ between different productions). You could change the name manually in the supermarket...

The conceptual problem is that you would have to be able to specifiy which variable substitution should not be persisted, so it is resolved over and over.
A dirty hack might be naming conventions "city_transient" as variable name to indicate that this variable should be forgotten on a "resetTransientVariables" call.

Reply:
I too think we should think about additional flags:
- POOL_RESETS_VARIABLES  (all)
- POOL_RESETS_TRANSIENT_VARIABLES (only "transient" ones)

Regarding variable naming we could either name them "normal" an add options (a bitmask) for each variable defined (so a second "collection" next to it)
or require a kind of special naming scheme (which is "easier" to do but looks a bit more dirty/hackish).

Reply:
This is a very special case, because usually you want consistent substitutions. POOL_RESETS_VARIABLES also is not quite the right naming. The template's variables are always immediately reset once the script is created (and the script itself does not contain variables anymore).
So transient may not be the correct word; rather it should probably be something like "conceptVariable_city", indicating that it is a variable that should only be resolved once the concept is created from the script.

With the expression refactoring, I could think of another apporach - a function that says "retain on script creation", so the result still contains the "inner variable", on concept creation this variable could be resolved (and always reset; because otherwise you could have resolved it already).